docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,113 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Ledger API Test Tool
+####################
+
+The Ledger API Test Tool is a command line tool for testing the correctness of
+implementations of the :doc:`Ledger API
+</app-dev/ledger-api-introduction/index>`, i.e. DAML ledgers. For example, it
+will show you if there is consistency or conformance problem with your
+implementation.
+
+Its intended audience are developers of DAML ledgers, who are using the
+:doc:`DAML Ledger Implementation Kit </ledger-integration-kit/index>` to develop
+a DAML ledger on top of their distributed-ledger or database of choice.
+
+Use this tool to verify if your Ledger API endpoint conforms to the :doc:`DA
+Ledger Model </concepts/ledger-model/index>`.
+
+Please note that currently the tool is in ALPHA status:
+
+- ALPHA: contains a subset of the tests we use internally to test our ledger
+  implementations.
+- BETA: will be reached once we extended the coverage to include the full set of
+  our test as per <https://github.com/digital-asset/daml/issues/146>.
+
+Downloading the tool
+====================
+
+TODO: To be defined in <https://github.com/digital-asset/daml/issues/142>.
+
+Extracting ``.dar`` file required to run the tests
+======================================================
+
+Before you can run the Ledger API test tool on your ledger, you need to load a
+specific set of DAML templates onto your ledger.
+
+#. To obtain the corresponding ``.dar`` file, run:
+
+   ``ledger-api-test-tool -x``
```

Comment:
`-x` is a pretty cryptic flag. Does it have a fuller, clearer name you can use instead?

Reply:
Replaced with the full name `--extract` here and below